kyolo/utils/bounding_box_utils.py

```diff
@@ -0,0 +1,92 @@
+from typing import List, Tuple
+
+from keras import KerasTensor, Model, ops
+
+
+def generate_anchors(image_size: List[int], strides: List[int]):
```

Comment:
better to use Tuple[int,int] as typehint for image_size. and shall we allow single int?